jsonrpc/src/jsonrpc.ml

```diff
@@ -38,10 +38,24 @@ module Constant = struct
 end
 
 module Message = struct
+  module Structured = struct
```

Comment:
How about `Params` instead to use the same terminology as VSCode?

Reply:
The name `Structured` comes from the jsonrpc [spec](https://www.jsonrpc.org/specification#parameter_structures), so it's more descriptive of the value used.